spec/spec_helper.cr

```diff
@@ -5,6 +5,8 @@ require "./support/**"
 
 include LuckyFlow::Expectations
 
+LuckyFlow::Spec.setup
```

Comment:
This sets up the lucky flow driver if one is requested through tags on the spec, and then resets back to the default driver after the spec is run.

It also has a callback for shutting down the drivers once all specs finish.

Reply:
I wonder if there would be a way to force people to call this... Or like, you try to run some specs, but you forgot to add this so you get a compile-time error telling you to add it 🤔

Reply:
I don't know about "force", but another option would be to have `luckyflow/spec` unrequired by the library but requires luckyflow and does the Spec setup right in the file. Then we document that in order to use the library in your specs, you must `require "lucky_flow/spec"` after you `require "spec"` and it will just work. It could potentially be a pattern that we could support spectator in the future if we want to (just `require "lucky_flow/spectator"`). Thoughts on that?